woocommerce/wc-admin
# Categories table

Fixes #897.

Adds a Table to the _Categories_ report.

Please, notice the data is loaded from SwaggerHub, for now.

### Screenshots
![image](https://user-images.githubusercontent.com/3616980/48924191-3f964180-ee7b-11e8-9271-9c000211f100.png)

### Detailed test instructions:
- Go to the _Categories_ report.
- Verify there is a table and interact with it.
